package.json

```diff
@@ -77,22 +77,22 @@
     "cli-table3": "^0.6.0",
     "command-exists": "^1.2.7",
     "contentful-export": "^7.11.5",
-    "contentful-import": "^7.9.23",
+    "contentful-import": "^8.1.12",
     "contentful-management": "^7.3.1",
-    "contentful-migration": "^4.2.0-experimental.1",
+    "contentful-migration": "4.2.0-experimental.1",
```

Comment:
Just a heads up. The experimental branch of `contentful-migration` needs to be synced with master as well. Maybe that will bump the version.